plot_pixel_values.py

```diff
@@ -22,24 +24,70 @@ def get_opt():
     parser.add_argument('--fit-scaling',
                         action='store_true',
                         help='Fit the dark and scale factor (default=False)')
-
+    parser.add_argument('--plot-fits',
+                        action='store_true')
     parser.add_argument('--n-brightest',
-                        default=128,
+                        default=64,
                         type=int,
-                        help='Plot the N brightest (either 128 or n**2)')
+                        help='Plot the N brightest (either 64 or n**2)')
 
     args = parser.parse_args()
     return args
 
 opt = get_opt()
 
+
+def scale_model(pars, t_ccd):
+    scale, y_const, x_const = pars
+    return 1 / np.exp(np.log(scale) / 4.0 * (t_ccd - x_const))
```

Comment:
Right.  I just couldn't recall if using -19 as our reference temp was based on reading off something like 0.70 from the standard curve at -19C or if we'd really fit that (in which case I wondered if letting x_const/T_CCD_REF it float and refitting on some other data set might be helpful).  I couldn't find our actual fits for the 0.70 , 0.68 scale factor.